apiserver/facades/agent/uniter/uniter.go

```diff
@@ -984,6 +984,111 @@ func (u *UniterAPI) WatchActionNotifications(args params.Entities) (params.Strin
 	return common.WatchActionNotifications(args, canAccess, watchOne), nil
 }
 
+// WatchActionNotifications returns a StringsWatcher for observing the status of an Upgrade Series
```

Comment:
doc string: WatchActionNotifications creates a NotifyWatcher

Reply:
Bah, the copy pasta monster.